src/main/resources/archetype-resources/src/assembly/policy-assembly.xml

```diff
@@ -37,6 +37,19 @@
             <directory>${json-schema-generator-maven-plugin.outputDirectory}</directory>
             <outputDirectory>schemas</outputDirectory>
         </fileSet>
+        <fileSet>
+            <directory>src/main/resources/schemas</directory>
+            <outputDirectory>schemas</outputDirectory>
+        </fileSet>
+
+        <!-- Include the Policy documentation -->
+        <fileSet>
+            <directory>${basedir}</directory>
+            <includes>
+                <include>README.adoc</include>
+            </includes>
+            <outputDirectory>docs</outputDirectory>
+        </fileSet>
```

Comment:
What about adding a fileset for the icon as well? 

See: https://github.com/gravitee-io/gravitee-policy-mock/blob/master/src/assembly/policy-assembly.xml#L49

And changing the plugin.properties file to add the category and icon?

See: https://github.com/gravitee-io/gravitee-policy-mock/blob/master/src/main/resources/plugin.properties#L7

Reply:
Including fileset for icon and a default category is no problem. I have those updates working.

However, adding icon to `plugin.properties` will cause a `An error occurs while trying to get plugin icon for plugin beta-archetype-policy` error unless some icon file is included. I have tried with:

```
# dummy file
icon=icon.svg
# no file
icon=
```

How would you like me to handle this? A few options could be:

1. Include the mock icon from the `gravitee-policy-mock` project you linked, so it's included with generation
2. Update the readme to include an `adding custom icon` section.

Reply:
Ah yes, you are right.
I would go for 2, and maybe we can just remove it from plugin.properties file to avoid failures (not a mandatory value).

Reply:
Sounds good. I debated leaving it in but commented out (`#icon`), but have removed it and added some additional documentation.